packages/OpenSqueakMap.package/OSMChunkManager.class/instance/updateChunks.st

```diff
@@ -0,0 +1,11 @@
+events
+updateChunks
+
+	| newChunk index |
+	self class cachedChunks do: [:chunk | 
+		index := chunk index.
+		(self chunks includesKey: index) ifFalse: [
```

Comment:
evtl. Formatierung fixen (hier ist es gerade im JavaScript-Style; `ifFalse: [` müsste auf eine neue Zeile.

(betrifft viele Stellen in diesem PR)